src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1389,6 +1394,23 @@ public void TestCeil()
             }
         }
 
+
+        [Test]
+        public void TestCeil_DifferentCultureDecimalSeparator()
+        {
+            using (CultureHelper.SetCulture("el-GR"))
+            {
+                Helper.AssertTemplateResult("2", "{{ 1,2 | ceil }}");
+                Helper.AssertTemplateResult("2", "{{ 2,0 | ceil }}");
+                Helper.AssertTemplateResult("184", "{{ 183,357 | ceil }}");
+                Helper.AssertTemplateResult("4", "{{ \"3,5\" | ceil }}");

```

Comment:
I would suggest an additional test for thousands separators for both cultures. e.g. 1.234.567,89 should parse for el-GR and 1,234,567.89 for en.